Makefile

```diff
@@ -30,16 +30,15 @@ BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
 # This variable is used to construct full image tags for bundle and catalog images.
 #
 # For example, running 'make bundle-build bundle-push catalog-build catalog-push' will build and push both
-# example.com/gitops-primer-bundle:$VERSION and example.com/gitops-primer-catalog:$VERSION.
-IMAGE_TAG_BASE ?= quay.io/migtools/gitops-primer
-# IMAGE_TAG_BASE ?= quay.io/screeley44/gitops-primer
+# example.com/snapshot-operator-bundle:$VERSION and example.com/snapshot-operator-catalog:$VERSION.
+IMAGE_TAG_BASE ?= quay.io/kubesaw/snapshot-operator
 
 # BUNDLE_IMG defines the image:tag used for the bundle.
 # You can use it as an arg. (E.g make bundle-build BUNDLE_IMG=<some-registry>/<project-name-bundle>:<tag>)
 BUNDLE_IMG ?= $(IMAGE_TAG_BASE)-bundle:v$(VERSION)
 
 # Image URL to use all building/pushing image targets
-IMG ?= quay.io/migtools/gitops-primer:v0.0.11
+IMG ?= quay.io/kubesaw/snapshot-operator:v0.0.11
```

Comment:
are we going to use version tags or just `latest`?

Reply:
i'm not 100% sure, but will just set to `latest` for now